src/t8_cmesh/t8_cmesh_geometry.cxx

```diff
@@ -47,9 +47,9 @@ t8_cmesh_set_tree_geometry (t8_cmesh_t cmesh, const t8_gloidx_t gtreeid, const t
 {
   T8_ASSERT (t8_cmesh_is_initialized (cmesh));
   /* Add the hash of the geometry as an attribute to the tree. */
-  size_t hash = geom->t8_geom_get_hash ();
-  t8_cmesh_set_attribute (cmesh, gtreeid, t8_get_package_id (), T8_CMESH_GEOMETRY_ATTRIBUTE_KEY, &hash, sizeof (size_t),
-                          0);
+  t8_geometry_hash_t hash = geom->t8_geom_get_hash ();
```

Comment:
We decided to not name our types using `_t` since it was not really clear if it is a typedef or type and it is also reserved for POSIX sytsems

Reply:
Thanks for catching this.

I added this section to our dev guidelines: https://github.com/DLR-AMR/t8code/wiki/Coding-Guideline#data-types-and-classes